gateway-messages/src/sp_to_mgs/sp5_details.rs

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::tlv;
+use hubpack::SerializedSize;
+use serde::{Deserialize, Serialize};
+
+/// Most recent POST code seen by the sequencer FPGA
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, SerializedSize)]
+pub struct LastPostCode(pub u32);
```

Comment:
oh, another thing that occurred to me just now: i wonder if this message ought to include some kind of timestamp? we could either just include the total Hubris uptime in milliseconds (which would be very easy to add) or perhaps a time since the system was commanded to A0 (which could be more annoying)...or even a relative timestamp since the POST code last changed? this could be a bunch of extra work, but perhaps we might want to be able to look at how long we've been in a particular boot state?

it's fine to punt on that in this PR --- I know we have also discussed the idea of a ringbuf of the last _n_ POST codes, so if we go back and implement that, we could just expose timestamps in *that* interface...